plugins/module_utils/data_set.py

```diff
@@ -17,33 +17,30 @@
 import tempfile
 import traceback
 from os import path, walk
-from string import ascii_uppercase, digits
 from random import sample
+from string import ascii_uppercase, digits
+
 # from ansible.module_utils._text import to_bytes
 from ansible.module_utils.common.text.converters import to_bytes
-from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.ansible_module import (
-    AnsibleModuleHelper,
-)
-from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.import_handler import (
-    MissingImport,
-    ZOAUImportError,
-)
-
 from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import (
-    better_arg_parser,
-    mvs_cmd,
-)
+    better_arg_parser, mvs_cmd)
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.ansible_module import \
+    AnsibleModuleHelper
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.import_handler import (
+    MissingImport, ZOAUImportError)
 
 try:
     from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import vtoc
 except ImportError:
     vtoc = MissingImport("vtoc")
 
 try:
-    from zoautil_py import datasets, exceptions
+    from zoautil_py import datasets, exceptions, gdgs
 except ImportError:
     datasets = ZOAUImportError(traceback.format_exc())
     exceptions = ZOAUImportError(traceback.format_exc())
+    gdgs = ZOAUImportError(traceback.format_exc())
+    Dataset = ZOAUImportError(traceback.format_exc())
```

Comment:
I don't intent to make code changes that are not bugfixes at this step, this would involve testing this module_util dependencies all over again which are many, I rather create a GH issue to address this in development branch.